README.md

```diff
@@ -7,9 +7,9 @@ Inder Khera, Jenny Zhang, Jessica Kuo, Javier Martinez (alphabetically ordered)
 ## About
 
 In this study, we aim to develop a classification model using the logistic regression (LR) algorithm to predict whether a patient is expected to have diabetes or not. 
-Our final model performed decent on an unseen test dataset, achieving an overall accuracy of 0.75. Out of 218 test cases, the model correctly identified 164. 
-However, it made 46 incorrect predictions, of which, 19 are false positives - incorrectly classifying non-diabetic subjects to diabetic- 
-and 35 are false negatives - fail to diagnose diabetes when the patient is actually diabetic. 
+Our final model performed decent on an unseen test dataset, achieving an overall accuracy of 0.80. Out of 101 test cases, the model correctly identified 81. 
```

Comment:
I think we can include both the results from train and test. In fact, can we do something similar to the below from Tiff's example? I think all the write ups can be grabbed somewhere in Quarto, just need to concatenate.

```
Here we attempt to build a classification model using the k-nearest neighbours algorithm which can use breast cancer tumour image measurements to predict whether a newly discovered breast cancer tumour is benign (i.e., is not harmful and does not require treatment) or malignant (i.e., is harmful and requires treatment intervention). Our final classifier performed well on an unseen test data set, with the F2 score, where beta = 2, of 0.97 and an overall accuracy calculated to be 0.87. On the 171 test data cases, it correctly predicted 157. Nine mistakes were predicting a benign tumour as malignant, while 4 mistakes where predicting a malignant tumour as benign. This is somewhat promising for implementing this in the clinic as false positives are less harmful than false negatives. Although they could theoretically cause the patient to undergo unnecessary treatment if the model is used as a decision tool, it is likely that the model is used for initial screening and that there will be a follow up appointment and further testing until treatment commences. However, the observation of even 4 mistakes predicting a malignant tumour as benign is concerning. As such, we believe further development of this model is needed for it to have clinical utility. Research to improve the model performance and understand the characteristics of incorrectly predicted patients is recommended.
```